modules/server/src/main/scala/com/itforelead/smspaltfrom/services/SMSTemplates.scala

```diff
@@ -0,0 +1,37 @@
+package com.itforelead.smspaltfrom.services
+
+import cats.effect.{Resource, Sync}
+import cats.implicits._
+import com.itforelead.smspaltfrom.domain.{ID, SMSTemplate}
+import com.itforelead.smspaltfrom.domain.SMSTemplate.CreateSMSTemplate
+import com.itforelead.smspaltfrom.domain.types.TemplateId
+import com.itforelead.smspaltfrom.effects.GenUUID
+import skunk.{Session, Void}
+
+trait SMSTemplates[F[_]] {
+  def create(form: CreateSMSTemplate): F[SMSTemplate]
+  def templates: F[List[SMSTemplate]]
+}
+
+object SMSTemplates {
+  def apply[F[_]: GenUUID: Sync](implicit
+    session: Resource[F, Session[F]]
+  ): SMSTemplates[F] =
+    new SMSTemplates[F] with SkunkHelper[F] {
+
+      import com.itforelead.smspaltfrom.services.sql.SMSTemplateSql._
+
+      def create(form: CreateSMSTemplate): F[SMSTemplate] = {
```

Comment:
Shu method ni boshqacharoq yozisham mumkin yani for siz.
```suggestion
      def create(form: CreateSMSTemplate): F[SMSTemplate] =
         ID.make[F, TemplateId].flatMap { id =>
           prepQueryUnique(insert, SMSTemplate(id, form.text, form.active))
         }
```